### PR TITLE
Remove all links to migration page and update nav page in Core docs 2.1.5 - closes #603 

### LINKED ISSUE #603: Remove all links to migration page and update nav page in Core docs 2.1.5
Remove all links to Migration page and update nav page. (leave migration page)

Parent issue #591

### PR DESCRIPTION
**Pull request by chris529**
closes #603

### CODE REVIEW

**Review by Tschakki — CHANGES_REQUESTED** [review]
Please resolve the conflicts first.

**Review by Tschakki — CHANGES_REQUESTED** [review]
Please focus in the PR on the issue you are solving. There are many unrelated changes in this PR.

File: modules/ROOT/pages/management/docker.adoc [hunk 1]
```diff
@@ -6,12 +6,11 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 :page-next-title: Configuration
 :page-previous-title: Docker installation
-
+:url_docker_compose_ports: https://docs.docker.com/compose/reference/port/
+:url_docker_compose_compose: https://docs.docker.com/compose/compose-file/#environment
 :url_docker_compose_exec: https://docs.docker.com/compose/reference/exec/
 :url_lisk_snapshots: http://snapshots.lisk.io
-
 :url_config: management/configuration.adoc
-:url_config_cl: reference/config.adoc#clo
```
Comment (Tschakki) [inline modules/ROOT/pages/management/docker.adoc]: Why are you removing this link?

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/management/docker.adoc [hunk 2]
```diff
@@ -6,11 +6,10 @@ Mona Bärenfänger <mona@lightcurve.io>
 :page-previous: /lisk-core/setup/docker.html
 :page-next-title: Configuration
 :page-previous-title: Docker installation
-
+:url_docker_compose_ports: https://docs.docker.com/compose/reference/port/
+:url_docker_compose_compose: https://docs.docker.com/compose/compose-file/#environment
 :url_docker_compose_exec: https://docs.docker.com/compose/reference/exec/
 :url_lisk_snapshots: http://snapshots.lisk.io
-
-:url_config: management/configuration.adoc
```
Comment (Tschakki) [inline modules/ROOT/pages/management/docker.adoc]: Why are you removing this link?

**Review by Tschakki — CHANGES_REQUESTED** [review]

**Comment by Tschakki** [comment]
Also, please fix the conflicts

File: modules/ROOT/pages/management/docker.adoc (hunk 2 above)
Comment (chris529) [reply modules/ROOT/pages/management/docker.adoc]: OK, it is now restored.

**Review by Tschakki — APPROVED** [review]
Thank you!

**Pull request merged**